src/actinia_core/endpoints.py

```diff
@@ -201,13 +201,25 @@ def create_core_endpoints():
     flask_api.add_resource(TokenCreationResource, '/token', )
     flask_api.add_resource(APIKeyCreationResource, '/api_key', )
     flask_api.add_resource(APILogResource, '/api_log/<string:user_id>')
+
     # Resource management
+    """
+    The endpoint '/resources/<string:user_id>/<string:resource_id>' has to
+    different answerd depending on the resource_id. If the resoucre_id starts
+    with 'resouce-id_' the latest iteration of the resouce is given back.
+    If the resocue_id is only the id than all iterations of the resource are
+    given in the response.
+    """
```

Comment:
`has to different answerd` = has a different answer?
Also some typos in this block: `resoucre_id`, `'resouce-id_'`, `resouce`, `resocue_id` and `is only the id than` -> then